nusight2/src/client/components/localisation/view.tsx

```diff
@@ -201,3 +202,19 @@ const FieldLineDots = ({ model }: { model: LocalisationModel }) => (
     )}
   </>
 );
+
+const Ball = ({ model }: { model: LocalisationModel }) => (
+  <>
+    {model.robots.map(
+      (robot) =>
+        robot.visible && (
+          <object3D key={robot.id}>
+            <mesh position={robot.ball.toArray()}>
+              <sphereBufferGeometry args={[robot.ball.z, 20, 20]} />
+              <meshStandardMaterial color="#1E90FF" />
```

Comment:
Any reason for this particular colour? Should it be white?

Reply:
I thought the balls were orange

Reply:
Just chose a random colour. White is probably a better option.

Reply:
Hmm, white doesn't work well with lighting, orange looks good